commands/box/transaction/root.go

```diff
@@ -263,11 +263,16 @@ func createRawTransaction(cmd *cobra.Command, args []string) {
 		return
 	}
 	defer conn.Close()
-	height, err := rpcutil.GetBlockCount(conn)
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancel()
+	req := &rpcpb.GetCurrentBlockHeightRequest{}
+	client := rpcpb.NewContorlCommandClient(conn)
+	resp, err := client.GetCurrentBlockHeight(ctx, req)
 	if err != nil {
 		fmt.Println(err)
 		return
 	}
+	height := resp.Height
```

Comment:
no need to use height variable. height could be 0 to pass param in CreateRawTransaction